source/application/AMDSCentralServerSGMAmptek.cpp

```diff
@@ -4,28 +4,35 @@
 
 #include "Connection/AMDSThreadedTCPDataServer.h"
 #include "ClientRequest/AMDSClientRequest.h"
+#include "ClientRequest/AMDSClientConfigurationRequest.h"
 #include "DataElement/AMDSThreadedBufferGroup.h"
 #include "Detector/Amptek/AmptekSDD123ConfigurationMap.h"
 #include "Detector/Amptek/AmptekSDD123DetectorManager.h"
 #include "Detector/Amptek/AmptekSDD123ServerGroup.h"
 #include "Detector/Amptek/AmptekSDD123Server.h"
-#include "Detector/Amptek/SGM/AmptekSDD123DetectorGroupSGM.h"
+#include "Detector/Scaler/AMDSScalerCommandManager.h"
 #include "Detector/Scaler/AMDSScalerConfigurationMap.h"
 #include "Detector/Scaler/AMDSScalerDetector.h"
-#include "Detector/Scaler/AMDSScalerDetectorManager.h"
+#include "Detector/Scaler/AMDSScalerDetectorServer.h"
+#include "Detector/AMDSDetectorServer.h"
+
 #include "util/AMErrorMonitor.h"
 #include "util/AMDSRunTimeSupport.h"
 
+#include "Detector/Amptek/SGM/AmptekCommandManagerSGM.h"
+#include "Detector/Amptek/SGM/AmptekSDD123DetectorGroupSGM.h"
+
 AMDSCentralServerSGMAmptek::AMDSCentralServerSGMAmptek(QObject *parent) :
 	AMDSCentralServer(parent)
 {
-	maxBufferSize_ = 1 * 60 * 1000; // 20 minuntes
+	maxBufferSize_ = 1 * 60 * 1000; // 20 minuntes data recording
```

Comment:
Are we still wanting 20 minutes of data recording?  Because this is only 1 minute.